src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -41,10 +43,14 @@
     public void before() throws IOException {
         // create event-type and subscribe to it
         eventType = createEventType();
-        subscription = createSubscription(ImmutableSet.of(eventType.getName()));
+        final SubscriptionBase subscription = randomSubscription()
+                .withEventType(eventType.getName())
+                .withStartFrom(BEGIN)
+                .buildSubscriptionBase();
+        this.subscription = createSubscription(subscription);
     }
 
-    @Test(timeout = 30000)
```

Comment:
`I understood why you removed this timeout` I don't :)
I think I did it accidentally. The purpose of this timeout is to limit test run time. The test can stuck not only in the place if `waitFor` but also for example when `commitCursors` method is called (and there we don't have a timeout)
So I would prefer return 30 sec timeout back.